package.json

```diff
@@ -54,7 +54,7 @@
     "sirv": "^1.0.17",
     "ts-jest": "^27.0.4",
     "ts-node": "^10.1.0",
-    "typescript": "^4.3.5",
+    "typescript": "4.3.5",
```

Comment:
You may need to use `yarn install --frozen-lockfile`
Please do not alter `package.json` within this PR

Reply:
sorry for the mess, should have read contributing.md earlier

Reply:
Please revert this change